docs/source/hardware/scavenger.rst

```diff
@@ -0,0 +1,69 @@
+
+.. _partition_names:
+
+**********
+Scavenger
+**********
+:sub:`Updated July, 2022`
+
+Introduction
+===================
+‘Scavenger’ is a new partition on discovery that allows the research community to use private resources 
+on the cluster when they are idle. ‘Scavenger’ has hardware not otherwise available to the general research 
+community and, in time, will double the resources to which users have access.
+
+When to use scavenger
+===================
+
+From best to less-good case scenarios:
+
+1. Code that can be check-pointed
+2. Jobs that fit on a single node
+3. Jobs that require multiple nodes (eg, MPI)
+4. When waiting is too hard
+
+How to use scavenger
+===================
+
+To use the ``scavenger`` partition, just add that to your partition list. As the partition list is a 
+comma-delimited list of values, ``srun --partition=short,scavenger`` is perfectly reasonable. For this example, 
+we will exclude partition ``short`` and focus on ``scavenger``. When you run the ``squeue`` command, you can see 
+that your job has been assigned to the scavenger partition::
+
+  [m.joshi@login-01 ~]$ srun  -p scavenger --pty /bin/bash
+  srun: job 23498584 queued and waiting for resources
+  srun: job 23498584 has been allocated resources
+
+  [m.joshi@d3110 ~]$ squeue -u m.joshi
+  JOBID    PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
+  23498592      ctbp     bash  m.joshi  R       0:07      1 d3110
+  23498584 scavenger     bash  m.joshi  R       2:19      1 d3110
+
+For the example provided on our `checkpointing <https://rc-docs.northeastern.edu/en/latest/best-practices/checkpointing.html?highlight=array#gromacs-checkpointing-example>`_ page, you can use the scavenger partition as::
+
+ #!/bin/bash
```

Comment:
Can we block-quote this script so it appears as such?

Reply:
@dhml It is currently block quoted in a way. You can see how it looks like here https://northeastern-university-rc-public-documentation--19.com.readthedocs.build/en/19/ and navigating to 'Hardware on Discovery' > Scavenger (The link will be under 'All checks have passed' > Show all checks > Details if you scroll down on the PR page). Is this what you meant?

Reply:
@ShobanaSekar, look good and thanks for the pointer!  I'm not yet that familiar with Sphinx and `*.rst` files.